tests/unit/meta/test_meta.py

```diff
@@ -1110,6 +1110,48 @@ def test_snap_hooks_overwrite_part_hooks(self):
             FileContains("from snap"),
         )
 
+    def test_hooks_with_command_chain_write_stub(self):
+        self.config_data["hooks"] = {
+            "configure": {"command-chain": ["fake-hook-command-chain"]}
+        }
+
+        self.generate_meta_yaml()
+
+        self.assertThat(os.path.join(self.hooks_dir, "configure"), FileExists())
+
+        # The hook should be empty, because the one in snap/hooks is empty, and
+        # no wrapper is generated (i.e. that hook is copied to both locations).
+        self.assertThat(
+            os.path.join(self.hooks_dir, "configure"), FileContains("#!/bin/sh\n")
+        )
+
+    def test_snap_hooks_stubs_not_created_stubs_from_command_chain(self):
+        self.config_data["hooks"] = {
+            "install": {"command-chain": ["fake-hook-command-chain"]}
+        }
+
+        # Setup a prime/snap directory containing a hook.
+        part_hook = os.path.join(self.prime_dir, "snap", "hooks", "install")
+        _create_file(part_hook, content="from part", executable=True)
+
+        # Now write the snap directory, and verify that the snap hook overwrote
+        # the part hook in both prime/snap/hooks and prime/meta/hooks.
+        self.generate_meta_yaml()
+
+        self.assertThat(
+            os.path.join(self.hooks_dir, "install"),
```

Comment:
Should we verify the snap/hooks/intall is expected?

Reply:
there's already a test for that above or below :-)

Reply:
on a second look, snap/hooks/install is what I setup in the test setup, is the expactation to always test that what was setup still exists after excercising anything?

Reply:
I was just thinking of validating that this logic didn't break the snap/hooks.  But looking back at the code, seems unnecessary.